modules/affile/affile-source.c

```diff
@@ -253,6 +253,43 @@ affile_sd_construct_proto(AFFileSourceDriver *self, gint fd)
     }
 }
 
+static void
+affile_sd_reset_file_state(PersistState *state, const gchar *old_persist_name)
+{
```

Comment:
hmm.. if this is an AFFile method, please pass a self argument around, maybe then we wouldn't need to pass state or old_persist_name as they could be formatted.

Also, please use a more specific term than "reset", this seems to drop the old position information for this file. Maybe we would be better of by adding a _delete() operation to persist_state instead of playing dirty tricks of renaming entries and not caring them about later?